### PR TITLE
Feature/misc bugs etc

### PR DESCRIPTION
**Pull request by jwhaney**
index map on load fit bounds to marker clusters; issue #72; instruction/about back to top link fix; issue #131;

### CODE REVIEW

File: lakegallery/map/static/map/style.css [hunk 1]
```diff
@@ -485,7 +486,16 @@ ol > img {
...
 .modal-body > .close {
+	width: 30px;
+	height: 30px;
+	top: 20px;
+	right: 20px;
 	cursor: pointer;
+	position: absolute;
+	background-color: white;
+	border-radius: 15px;
+  color: black;
```
Comment (adambreznicky) [inline lakegallery/map/static/map/style.css]: align?

File: lakegallery/map/templates/map/story.html [hunk 2]
```diff
@@ -24,31 +26,31 @@ <h2 class="lakeTitle">{{ lake }}</h2>
             <nav id="navbar-contentPanel" class="navbar navbar-light bg-light">
               <ul id="storyNav" class="nav nav-pills">
                 <li class="nav-item">
-                  <a class="nav-link active" href="#Overview">Overview</a>
+                  <a class="nav-link active" href="#Overview" tabindex="0">Overview</a>
```
Comment (adambreznicky) [inline lakegallery/map/templates/map/story.html]: accessibility tabindexes.... very cool!

File: lakegallery/static/map/style.css [hunk 3]
```diff
@@ -485,7 +486,16 @@ ol > img {
...
 .modal-body > .close {
+	width: 30px;
+	height: 30px;
+	top: 20px;
+	right: 20px;
 	cursor: pointer;
+	position: absolute;
+	background-color: white;
+	border-radius: 15px;
+  color: black;
```
Comment (adambreznicky) [inline lakegallery/static/map/style.css]: align?

File: lakegallery/map/static/map/style.css (hunk 1 above)
Comment (jwhaney) [reply lakegallery/map/static/map/style.css]: i saw that. not sure why it shows that way. my style sheet code looks clean.
image

Comment (adambreznicky) [reply lakegallery/map/static/map/style.css]: oh, that is weird. i approve!

**Pull request merged**